classes/module/Module.php

```diff
@@ -866,6 +889,21 @@ public function disableDevice($device)
         return true;
     }
 
+    public function isDeviceEnabled(int $device): bool
+    {
+        $query = new DbQuery();
+        $query->select('enable_device');
+        $query->from('module_shop', 'ms');
+        $query->where('ms.id_module = ' . (int) $this->id);
+
+        $row = Db::getInstance()->getRow($query);
+        if (!is_array($row)) {
+            return false;
+        }
+
+        return $row['enable_device'] & $device;
```

Comment:
```suggestion
        return (bool) ($row['enable_device'] & $device);
```

Otherwise it returns an int :thinking: